File: 0122-best-time-to-buy-and-sell-stock-ii/0122-best-time-to-buy-and-sell-stock-ii.java
class Solution {
    public int maxProfit(int[] prices) {
        int pro = 0;
        int n = prices.length;
        for(int i=1;i<n;i++){
            if(prices[i]>prices[i-1]){
                pro+=prices[i]-prices[i-1];
            }
        }
        return pro;
        
    }
}

File: 0122-best-time-to-buy-and-sell-stock-ii/0122-best-time-to-buy-and-sell-stock-ii.py
class Solution(object):
    def maxProfit(self, prices):
        pro = 0;
        n = len(prices);
        for i in range(1,n):
            if prices[i] > prices[i-1]:
                pro+=prices[i]-prices[i-1];
        return pro
        """
        :type prices: List[int]
        :rtype: int
        """
        